package.json

```diff
@@ -14,8 +14,10 @@
     "build": "lerna run build && npm run test",
     "prettier:check": "prettier --check .",
     "pretty": "prettier --write .",
+    "bump:alpha": "lerna version prerelease --preid alpha --exact --yes",
     "publish:alpha": "lerna publish from-package --dist-tag alpha --yes --no-verify-access",
-    "publish:production": "NEXT_PUBLIC_SEGMENT_KEY=ED3O1FVsa8DteNyRmAuDyMcPoftJ6VdP lerna publish from-package",
+    "bump:production": "lerna version --conventional-graduate --exact",
+    "publish:production": "NEXT_PUBLIC_SM_UI_SEGMENT_KEY=cGjidifKefYb6EPaGaqpt8rQXkv5TD6P PUBLIC_SM_INIT_SEGMENT_KEY=ED3O1FVsa8DteNyRmAuDyMcPoftJ6VdP lerna publish from-package",
```

Comment:
This is why I talked about having independent commands to deploy stuff.
Because the more we add env keys, the more this command is long.

Also, having a script that deploy 1 by 1 packages, would be good to have total control over the order of deployment and all.

Reply:
I think we can split it now